### PR TITLE
Bookstore: Initialize project with components

### PR DESCRIPTION
**Pull request by Timbar09**
In this PR. I initialized the bookstore app with components.

**Changes**

- Created and set up a React app
- Set up linters
- Created components
- Added a routing system

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @Timbar09, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has no professional README file. Kindly follow this [link](https://github.com/microverseinc/readme-template) to make your professional README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/routes/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  const bookStore = [
+    { id: '1', title: 'The Secret ot Life', author: 'Great Van Author' },
+    { id: '2', title: 'Magical Dancing Feet', author: 'Awesomenho Dos Santos' },
+    { id: '3', title: 'Precision and class', author: 'Luka Raadovic' },
+  ];
+  // const [bookStore, setBookStore] = useState([
+  //   { id: '1', title: 'The Secret ot Life', author: 'Great Van Author' },
+  //   { id: '2', title: 'Magical Dancing Feet', author: 'Awesomenho Dos Santos' },
+  //   { id: '3', title: 'Precision and class', author: 'Luka Raadovic' },
+  // ]);
```
Comment (VICTOR-CHUKWU) [inline src/routes/Books.jsx:14]: - [ ] [Optional] Kindly remove unused code

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Timbar09 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] good commit message

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Timbar09** [comment]
Thank you, @VICTOR-CHUKWU.

**Pull request merged**